src/mame/machine/lynx.cpp

```diff
@@ -2107,10 +2105,20 @@ DEVICE_IMAGE_LOAD_MEMBER(lynx_state::cart_load)
 	}
 	else
 	{
-		if (size > 0xffff) // 64,128,256,512k cartridges
-			m_granularity = size >> 8;
+		size_t audin_offset = image.get_software_region_length("audin_offset");
+		if (audin_offset)
+			m_audin_offset = audin_offset;
+
+		size_t granularity = image.get_software_region_length("granularity");
+		if (granularity)
+			m_granularity = granularity;
```

Comment:
This is not appropriate use for memory regions.  You should use a `feature` element in the software part to supply the value.